motr/m0crate/crate_io.c

```diff
@@ -343,7 +343,7 @@ int cr_namei_create(struct m0_workload_io *cwi,
 		    int                    obj_idx,
 		    int                    op_index)
 {
-	return m0_entity_create(NULL, &obj->ob_entity,
+	return m0_entity_create(&cwi->cwi_pool_id, &obj->ob_entity,
```

Comment:
What if the pool is not specified in the yaml file - how will it work? Will it select the default pool the same way it works now?

Reply:
I think we need to check if this param is specified or not in configure file.
If not, we still pass NULL to the m0_entity_create().

Reply:
If the value if not passed in the .yaml file will `&cwi->cwi_pool_id == NULL` ?

Reply:
Of course not. This is the address of "cwi->cwi_pool_id", and it is a valid pointer.
We should do:
```
(m0_fid_is_valid(&cwi->cwi_pool_id)) ? &cwi->cwi_pool_id : NULL 
```